### PR TITLE
nano php8 changes

### PR DESCRIPTION
**Pull request by laylatichy**

### CODE REVIEW

**Comment by laylatichy** [comment]
@waghanza PR for php8 compatibility for my framework, if there is any more issues I'll see semaphore CI

**Comment by waghanza** [comment]
Is this version also compatible with php7 ?

**Comment by laylatichy** [comment]
nope, php: >=8.0

File: php/nano/config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 framework:
   gitlab: x.laylatichy.x/nano
-  version: 0.0.9
+  version: 1.0.8
```
Comment (waghanza) [inline php/nano/config.yaml]: could you please use `1.0` ?

**Comment by laylatichy** [comment]
@waghanza updated

**Review by waghanza — APPROVED** [review]

**Pull request merged**